src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,14 +29,16 @@
 
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
 
-
+  private final BallChucker9000 BallChucker9000Subsystem;
 
   /**
    * The container for the robot.  Contains subsystems, OI devices, and commands.
    */
   public RobotContainer() {
     // Configure the button bindings
     configureButtonBindings();
+    BallChucker9000Subsystem = new BallChucker9000(new WPI_VictorSPX(4), new VictorSP(0), new VictorSP(1),
```

Comment:
You should declare these motor controllers above so you can invert both flywheel motors and the indexer